tests/pre-build-contracts.ts

```diff
@@ -47,7 +47,10 @@ async function compile(name: string): Promise<Compiled> {
 }
 
 const main = async () => {
-  await Promise.all(Object.keys(contractSources).map(compile));
+  for (let name of Object.keys(contractSources)) {
+    console.log(`Compiling ${name}`);
+    await compile(name);
+  }
```

Comment:
Doesn't this switch from parallel compilation to serial compilation? Is that intentional?